Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: build run clean tidy test test-e2e test-integration test-contract test-all lint lint-fix record-api swagger
+.PHONY: build run clean tidy test test-e2e test-integration test-contract test-all lint lint-fix record-api swagger install-tools
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)